### PR TITLE
Migrate CI to GitHub Actions

### LINKED ISSUE #59: Migrate to GitHub Actions
travis-ci.org stopped testing PRs and snapshots. Migrate the CI to GitHub Actions

### PR DESCRIPTION
**Pull request by datho7561**
Closes #59

Signed-off-by: David Thompson <davthomp@redhat.com>

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: Tests
+on: [push, pull_request]
+jobs:
+  tests:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-latest]
```
Comment (fbricon) [inline .github/workflows/tests.yml:8]: if you're running on a matrix, why not include windows?

Comment (datho7561) [reply .github/workflows/tests.yml:8]: I'll try including it, and see if it works.

Comment (datho7561) [reply .github/workflows/tests.yml]: Unfortunately, the tests weren't written with Windows in mind. I can work on rewriting them to be cross platform. Trying the tests under Windows revealed some new `eslint` errors that I've fixed in the PR as well.

Comment (datho7561) [reply .github/workflows/tests.yml]: Okay, I've updated the tests to also run and work on Windows.

**Pull request merged**